pages/osx/terminal-notifier.md

```diff
@@ -0,0 +1,20 @@
+# terminal-notifier
+
+> Send macOS User Notifications.
+> More information: <https://github.com/julienXX/terminal-notifier>.
+
+- Send a notification:
+
+`terminal-notifier -message {{'TLDR rocks'}}`
+
+- Display piped data with a sound:
+
+`echo {{'Piped Message Data!'}} | terminal-notifier -sound {{default}}`
```

Comment:
```suggestion
`echo '{{Piped Message Data!}}' | terminal-notifier -sound {{default}}`
```

Reply:
I'm not sure I understand why... you could replace this by double quoted strings, some string without spaces (so no quotation needed) or as I often like to do, backslashes.

Anyway the user-provided value is the whole chain, isn't it?

Reply:
I'm moving the quotes outside of the placeholder marks (`{{ }}`). I think single and double quotes are both fine.

Reply:
@acuteenvy I'm sorry but I still have to ask why... It is not in the `CONTRIBUTING.md` nor in `style-guide.md`. We can see examples of both ways of doing. I feel legitimate to ask why the change.

I really would rather not change that if there is no strong reason...

Reply:
There are more pages with `'{{ }}'` than with `{{' '}}`:
```bash
$ rg "^\`.*'\\{\\{.*" -l pages/ | wc -l
119

$ rg "^\`.*\\{\\{'.*" -l pages/ | wc -l
31
```

I'll request a review from someone else. Let's see what other people think.

Reply:
Regarding this, 

This is currently how the page renders with quotes inside:

![image](https://github.com/tldr-pages/tldr/assets/26346867/b9fc7222-e7bb-4ed9-a840-6fce2250b5e6)

And this is how it will render with quotes outside:

![image](https://github.com/tldr-pages/tldr/assets/26346867/386c5f63-f7ee-4f6e-8f49-89fc57835305)

Both are fine as it doesn't affect the final output but only the colour scheme. **But placing quotes outside is the correct method if it isn't part of the placeholder .**

> There are more pages with '{{ }}' than with {{' '}}:

Yep, `{{' '}} isn't the intended behaviour unless it is part of the placeholder input. I am currently on my holiday, so I will try to go through these pages carefully and edit wherever possible.